cuegui/cuegui/plugins/LogViewPlugin.py

```diff
@@ -362,6 +362,9 @@ def __init__(self, parent=None):
         self._content_box.moveCursor(QtGui.QTextCursor.End)
         self._content_box.ensureCursorVisible()
 
+        self.highlighter = Highlighter(self._content_box.document())
+        print("highlight turned on")
```

Comment:
I think we can drop this `print`, or turn it into a logger message if it's important.